src/components/ItaliaTheme/Blocks/BandiSearch/DefaultFilters.js

```diff
@@ -197,59 +195,62 @@ const DefaultFilters = () => {
     },
     scadenza_filter: {
       label: intl.formatMessage(messages.scadenza_filter),
-      type: 'scadenza_filter',
+      type: 'date_filter',
       widget: {
-        component: DateFilter,
+        component: DateRangeFilter,
         props: {
           value: {
-            startDate: moment().startOf('day'),
-            endDate: moment().endOf('day'),
+            start: null,
```

Comment:
perchè qua hai cambiato il default? 
prima era impostato con default sulla data di oggi

Reply:
È una scelta fatta sulla base di feedback dei clienti. Inoltre, il comportamento del widget vecchio era considerato scomodo, alcuni proprio non lo capivano, perché la croce di reset riporta al default che era la data di oggi, invece di rimuovere il valore, quindi in pratica non era possibile NON filtrare per date se il filtro era impostato per il blocco.

Reply:
ah ok, meglio cosi allora